### PR TITLE
EPMRPP-60912 - Unification of all preloaders

### PR DESCRIPTION
**Pull request by chivekrodis**

### CODE REVIEW

File: app/src/pages/inside/common/durationBlock/durationBlock.jsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ export class DurationBlock extends Component {
     return (
       <Fragment>
         <span className={cx('in-progress')}>
-          <img src={InProgressGif} alt="In progress" />
+          <DottedPreloader color="black" />
```
Comment (KatsiarynaTatarynovich) [inline app/src/pages/inside/common/durationBlock/durationBlock.jsx:173]: Have you discussed the view with the designer?

**Review by KatsiarynaTatarynovich — CHANGES_REQUESTED** [review]

File: app/src/layouts/common/sidebar/userBlock/userBlock.jsx [hunk 2]
```diff
@@ -35,21 +35,24 @@ import styles from './userBlock.scss';
...
 
-const UserAvatar = ({ photoTimeStamp }) => (
+const UserAvatar = ({ photoTimeStamp, preloaderColor }) => (
   <div className={cx('avatar-wrapper')}>
     <Image
       className={cx('avatar')}
       src={URLS.dataPhoto(photoTimeStamp, true)}
       alt="avatar"
       fallback={DefaultUserImage}
+      preloaderColor={preloaderColor}
```
Comment (KatsiarynaTatarynovich) [inline app/src/layouts/common/sidebar/userBlock/userBlock.jsx:45]: Could you please explain why you decided to work with color through a parent if this method is not used anywhere else?

**Review by KatsiarynaTatarynovich — CHANGES_REQUESTED** [review]

File: app/src/layouts/common/sidebar/userBlock/userBlock.jsx (hunk 2 above)
Comment (chivekrodis) [reply app/src/layouts/common/sidebar/userBlock/userBlock.jsx:45]: `UserAvatar` uses in sidebar with tooltip we need here a white color preloader. Also, `UserAvatar` uses in pop-up by clicking on user avatar and here we need a charcoal color. That reason why I have needed lifting this props.

Comment (chivekrodis) [reply app/src/layouts/common/sidebar/userBlock/userBlock.jsx:45]: Duration block, preloader on admin panel for all users image, attachments with photos, issue tooltip are having a charcoal color.
UserAvatar has white color.

**Review by KatsiarynaTatarynovich — APPROVED** [review]

**Pull request merged**